xtask/src/utils.rs

```diff
@@ -377,8 +399,8 @@ pub(crate) fn whycheproof_import() -> Result<(), crate::DynError> {
 }
 
 pub(crate) fn track_language_changes(last_lang: &str) -> Result<(), crate::DynError> {
-    let last_config = format!("target/{}/LAST_LANG", TARGET_TRIPLE);
-    std::fs::create_dir_all(format!("target/{}/", TARGET_TRIPLE)).unwrap();
+    let last_config = format!("target/{}/LAST_LANG", TARGET_TRIPLE_RISCV32);
```

Comment:
Shouldn't this write the actual `target` here rather than always rv32?

Reply:
Good catch! I think the actual builder target isn't known yet at the time when this function is called in `main.rs`. I also don't think that this file has to be target-dependent and instead could be target-global and just reside in the root of `target` directory, but please correct me if I'm wrong.

I'm thinking of replacing it with the following:

```rust
pub(crate) fn track_language_changes(last_lang: &str) -> Result<(), crate::DynError> {
    let last_config = "target/LAST_LANG";
    let mut contents = String::new();
``` 

So the `LAST_LANG` will become global across targets and directory creation is omitted because `target/` should be 
 already created by a build caused by running `cargo xtask`.